stingray/pulse/search.py

```diff
@@ -12,16 +12,18 @@
 
 
 @jit(nopython=True)
-def _pulse_phase_fast(time, f, buffer_array):
+def _pulse_phase_fast(time, f, fdot, buffer_array):
     for i in range(len(time)):
-        buffer_array[i] = time[i] * f
+        buffer_array[i] = time[i] * f + 0.5 * time[i]**2 * fdot
         buffer_array[i] -= np.floor(buffer_array[i])
     return buffer_array
 
 
 def _folding_search(stat_func, times, frequencies, segment_size=5000,
-                    use_times=False, **kwargs):
-    stats = np.zeros_like(frequencies)
+                    use_times=False, fdots=0, **kwargs):
+
+    FF, FD = np.meshgrid(frequencies, fdots)
```

Comment:
Minor nitpicking: I think PEP8 reserves upper-case letters for classes rather than variables, if we want to be strict about following conventions.

Reply:
I was just super-lazy and used the same convention I found in the first example of `np.meshgrid` I've ever seen. True story.
Renamed.